vignettes/vignette.Rmd

```diff
@@ -1,64 +1,72 @@
 
-<!--# Would it be better to download directly from the API and show the first step as saving the file? -->
+We will turn to one of the largest sources of fossil occurrence data, the [Paleobiology Database](https://paleobiodb.org). We opted to download our data by completing the 'Download' form on the Paleobiology Database website. Practically, this uses the options entered into the form to create a call to the database's API (application programming interface), and then enacts this call. More information about the API service can be found [here](https://paleobiodb.org/data1.2/).
 
-We opted to download our data by completing the 'Download' form on the Paleobiology Database website. Practically, this uses the options entered into the form to create a call to the database's API (application programming interface), and then enacts this call. More information about the API service can be found [here](https://paleobiodb.org/data1.2/).
+For our dataset, we pulled all occurrences associated with the taxon name 'Crocodylia', dated to the 'Paleogene'. All other settings were left at default. We obtained the standard (or recommended) data fields by checking the box to "Include all output blocks whose names are boldfaced below", and obtained our data in the form of a '.csv' file. We will see later in the vignette what this means in terms of what is contained within the dataset.
 
-For our dataset, we pulled all occurrences associated with the taxon "Crocodylia", dated to the "Paleogene", in the form of a ".csv" file. All other settings were left at default. We will see later in the vignette what this means in terms of what is contained within the dataset.
+## Rule 2: Keep raw data raw
 
-For reproducibility, we want to make sure that we have a copy of the full dataset as initially downloaded - this is the "raw" data. Automatically, when we use the online form, there is an output option checked which is labelled 'Include metadata at the beginning of the output'. This ensures that the raw data file includes a metadata 'header', which contains useful information linked to the dataset.
+For reproducibility, we want to make sure that we have a copy of the full dataset as initially downloaded - this is the "raw" data. Automatically, when we use the online form, there is an output option checked which is labelled "Include metadata at the beginning of the output". This ensures that the raw data file includes a metadata 'header', which contains useful information linked to the dataset.
 
 We can load and view the metadata to see what it includes.
 
-```{r read_metadata}
+```{r read_metadata, message = FALSE}
 # Load data file
-metadata <- read.csv("../data/Paleogene_crocs.csv")
+metadata <- read_csv("data/Paleogene_crocs.csv")
```

Comment:
This might be highly dependent on where you are rendering this from, but in general we should use relative paths, which means we would need to go up a folder, first, right?

Reply:
I removed the `../` because it doesn't work for me, but I've put it back in now.

Reply:
Using `../` here doesn't work because we also define the root directory in the setup R chunk. So we either need to remove `..` here or in the setup chunk. If we ditch the subfolder "vignettes" and just have everything in the top-level directory, we don't need to worry about shifting up at all. If keeping the "vignettes" folder is preferred, I think leaving `..` in the setup chunk makes the most sense.

Reply:
Looks like "root.dir" should be set in a `knitr::opts_knit$set()` call, not a `knitr::opts_chunk$set()` call: https://yihui.org/knitr/options/#package-options. I think this is what was causing the issues for me/us.

Reply:
Indeed, adding this to the top and then removing ".." from the csv paths works for me:
```r
knitr::opts_knit$set(root.dir = '..')
```